packages/rsocket-core/src/RSocketMachine.js

```diff
@@ -784,13 +784,14 @@ class RSocketMachineImpl<D, M> implements RSocketMachine<D, M> {
   }
 
   _handleRequestChannel(streamId: number, frame: RequestChannelFrame): void {
+    const payload = this._deserializePayload(frame);
     const existingSubscription = this._subscriptions.get(streamId);
     if (existingSubscription) {
       //Likely a duplicate REQUEST_CHANNEL frame, ignore per spec
       return;
     }
 
-    const payloads = new Flowable(subscriber => {
+    const payloads = new Flowable<Payload<D, M>>(subscriber => {
```

Comment:
I guess this bug happened due to the missing type of this `Flowable` so add it now.